src/com/gfredericks/test/chuck/generators.cljc

```diff
@@ -78,7 +82,8 @@
               (let [pairs (core/partition 2 v1)
                     names (map first pairs)
                     gens (map second pairs)]
-                `(for [[~@names] (gen/tuple ~@gens)
+                `(for [[~@names]
+                       (~'gen/tuple ~@gens)
```

Comment:
Yes, you are correct. This is wrong, has to be fixed in a different way. We can't use the ns alias with quote-unquote, at least it should be the full ns name.

My worry is that we are going to play the "separate macros for clj and cljs" dance here too.